quint/test/types/constraintGenerator.test.ts

```diff
@@ -115,4 +115,58 @@ describe('ConstraintGeneratorVisitor', () => {
 
     assert.sameDeepMembers(Array.from(errors.values()), [error])
   })
+
+  function testArityError(expr: string, location: string, message: string) {
+    const defs = [expr]
```

Comment:
This is called `expr` but is expected to contain the text for a `def` 
```suggestion
  function testArityError(def: string, location: string, message: string) {
    const defs = [def]
```